recipes/cuda-cupti/meta.yaml

```
@@ -0,0 +1,177 @@
+{% set name = "cuda-cupti" %}
+{% set version = "12.0.90" %}
+{% set lib_version = "2022.4.0" %}
+{% set cuda_version = "12.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set target_name = "x86_64-linux" %}  # [linux64]
+{% set target_name = "ppc64le-linux" %}  # [ppc64le]
+{% set target_name = "sbsa-linux" %}  # [aarch64]
+{% set target_name = "x64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_cupti/{{ platform }}/cuda_cupti-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 3c93a6a8c5eab3a56c1a0a514c0ac7d374807c70a9551155f089588b6c5e1d10  # [linux64]
+  sha256: bbabb508eb4fb0f9d98ac3ab4a40d0278de41cef95139c1bfe6436b716dd6cfc  # [ppc64le]
+  sha256: 33b3b06cf9838fa3289c7e40f143a8f58b941ccad93f9d0640a29bb83aa2152e  # [aarch64]
+  sha256: b65a34a3715e3ef2b835bdd36c43b21c20800325be1eda1e84ad4d80ee837c92  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+test:
+  commands:
+    - test -L $PREFIX/lib/libcupti.so.{{ version.split(".")[0] }}                            # [linux]
+    - test -L $PREFIX/lib/libcupti.so.{{ lib_version }}                                      # [linux]
+    - test -L $PREFIX/targets/{{ target_name }}/lib/libcupti.so.{{ version.split(".")[0] }}  # [linux]
+    - test -f $PREFIX/targets/{{ target_name }}/lib/libcupti.so.{{ lib_version }}            # [linux]
+    - if not exist %LIBRARY_BIN%\cupti64_{{ version.split(".")[0] }}.dll exit 1              # [win]
```

This test appears to be [failing on CI]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=688664&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=2fb5b0a7-737c-5f5c-8f3f-f6db6174bacf&l=1670 )

Looking at the `redist`, it appears this is named differently. So guessing this test should be something like the change below. Though please let me know if I'm missing something

```suggestion
    - if not exist %LIBRARY_BIN%\cupti64_{{ lib_version }}.dll exit 1              # [win]
```